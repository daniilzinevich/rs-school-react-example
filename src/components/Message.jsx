import React from 'react';

const Message = ({ author, message }) => {
  return <div className="Message">
    <span>{author}</span>
    <span className="Text">{message}</span>
  </div>;
};

export default Message;
