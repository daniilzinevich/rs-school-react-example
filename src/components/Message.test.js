import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

import React from 'react';
import { shallow } from 'enzyme';
import Message from './Message'

describe('<Message />', () => {
  // wrong
  it('render correctly', () => {
    const wrapper = shallow(<Message author="author" message="test message" />);

    expect(wrapper.html()).toMatchSnapshot()
  })
})