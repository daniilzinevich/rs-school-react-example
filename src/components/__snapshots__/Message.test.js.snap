// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Message /> render correctly 1`] = `"<div class=\\"Message\\"><span>author</span><span class=\\"Text\\">test message</span></div>"`;
